Server/routes/Mobile/containers.js

```diff
@@ -34,18 +34,29 @@ router.get('/:containerId', async (req, res, next) => {
 
 router.get('/:containerId/articleslist/', async (req, res) => {
```

Comment:
You have not updated the related doc file

Reply:
done